designer/submodules/packages/components/src/scenarios/filters/filterListItemLabel.tsx

```diff
@@ -15,14 +15,14 @@ export function FilterListItemLabel({ name, icon }: { name: string; icon?: strin
             case "name":
                 return t("table.filter.sortBy.name", "Name");
             default:
-                return name;
+                return displayableName || name;
         }
     }, [name, t]);
 
     return icon ? (
-        <Stack direction="row" spacing={1} alignItems="center" justifyContent="space-between">
+        <Stack direction="row" spacing={1} alignItems="center" justifyContent="flex-start">
+            <NuIcon title={tooltip} src={icon} sx={{ fontSize: "1.2em", color: "inherit" }} />
```

Comment:
> i don't like how missing icon looks now. also icon nearby checkbox doesn't play nice.

Ok, I understand. I've discussed this layout with my team. I don't like missing icons either. I like icon nearby checkbox, it looks better when one of the statuses has much longer name (see attached images).

To move forward...

In https://github.com/TouK/nussknacker/pull/4132 all "something-is-wrong" statuses are combined into one "Problem", which is short name. Therefore it won't be a problem (hehe...) to display icons with alignment to the right. I'll go back to this layout because of those missing icons, that do not look good at all...

> one of the reasons these two items should not be near

That does not help with vertical alignment. Even when they are further apart you'll see that something is off. 

I tried wraping all 3 elements (checkbox, icon and text) in one Stack with alignment, with no effect. The best visual effect I got with paddingTop on `<span>Cancelled</span>` but that seems to be too brute solution... 

I see your opinion. Now, what is your recommendation?

Reply:
> Even when they are further apart you'll see that something is off.

![obraz](https://user-images.githubusercontent.com/2381222/227489921-acc46016-ac07-4a02-b7fc-024bfe7dff58.png)

Reply:
After rebase:
- all problem statuses combned into one
- long name "More than one deployment running" is not a problem
- missing icons are still a problem, it doesn't look good

![obraz](https://user-images.githubusercontent.com/2381222/227503229-55679d4d-d967-4666-a097-e04d8f1955d4.png)